pkgs/development/libraries/physics/cernlib/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, gfortran, gnumake, imake, makedepend, motif, xorg }:
+
+stdenv.mkDerivation rec {
+  version = "2006";
+  name = "cernlib-${version}";
+
+  src = fetchurl {
+    url = "http://cernlib.web.cern.ch/cernlib/download/${version}_source/tar/${version}_src.tar.gz";
+    sha256 = "0awla1rl96z82br7slcmg8ks1d2a7slk6dj79ywb871j2ksi3fky";
+  };
+
+  buildInputs = with xorg; [ gfortran gnumake imake makedepend motif libX11 libXft libXt ];
```

Comment:
This should also link to gfortran libraries.

I don't understand how cross-compilation works in nixpkgs. How for gmake parameter we will get a derivation with host-compatible "make" and for motif we will get the target-compatible libraries.